tests/tt_metal/tt_metal/perf_microbenchmark/1_compute_mm/kernels/bmm_large_block_zm_fused_bias_activation_copy.cpp

```diff
@@ -194,10 +198,6 @@ void MAIN {
                             #endif
                         #endif
 
-                        #if defined FP32_DEST_ACC_EN or defined PACKER_L1_ACC
-                            PACK((  pack_reconfig_data_format(mm_out_cb_id) ));
-                        #endif
-
```

Comment:
out of curiosity, is there a reason that configure data format needs to run before configure packer l1?

Reply:
I see, thanks